index.html

```diff
@@ -182,6 +182,10 @@
           be assigned the new type and the interaction model of the resource MUST be changed to the
           interaction model assigned to the new type by [[LDP]].
         </p>
+        <p>
+          A PUT request that creates a new resource MUST NOT, while so doing, change the containment
+          triples in any LDPC. ([[LDP]] 5.2.4.1 SHOULD becomes MUST)
```

Comment:
You know, I honestly don't understand what our pattern of use is with ! vs no-!. I am happy to use whatever you think is more appropriate.